ush/forecast_postdet.sh

```diff
@@ -162,19 +163,40 @@ EOF
         delimiter=""
         IAU_INC_FILES=""
         for iaufhr in "${iaufhrs[@]}"; do
-          if (( iaufhr == 6 )); then
-            inc_file="atminc.nc"
+          if [[ "${DO_JEDIATMVAR:-NO}" == "YES" ]] && [[ "${PDY}${cyc}" != "${SDATE}" ]]; then
```

Comment:
This comparison to `SDATE` gives me a pause.  Can we stage a zero-increment file for `SDATE`?